src/app/page.tsx

```diff
@@ -1,4 +1,4 @@
-import "glider-js/glider.min.css";
```

Comment:
Don't remove the glider css, it is for the news section carousel, keep it at the top of the document to make sure we can override it with global css